PinkSea.Frontend/src/intl/i18n.ts

```diff
@@ -8,46 +8,51 @@ import It from '@/intl/translations/it'
 import Sv from '@/intl/translations/sv'
 import Es from '@/intl/translations/es'
 import Bg from '@/intl/translations/bg'
+import Zh from '@/intl/translations/zh'
 
 export default {
   en: {
-    name: "English",
+    name: 'English',
     translation: En,
   },
   id: {
-    name: "Bahasa Indonesia",
+    name: 'Bahasa Indonesia',
     translation: Id,
   },
   pl: {
-    name: "Polski",
+    name: 'Polski',
     translation: Pl,
   },
   ja: {
-    name: "日本語",
+    name: '日本語',
     translation: Ja,
   },
   de: {
-    name: "Deutsch",
+    name: 'Deutsch',
     translation: De,
   },
   fr: {
-    name: "Français",
+    name: 'Français',
     translation: Fr,
   },
   it: {
-    name: "Italiano",
+    name: 'Italiano',
     translation: It,
   },
   sv: {
-    name: "Svenska",
+    name: 'Svenska',
     translation: Sv,
   },
   es: {
-    name: "Español",
-    translation: Es
+    name: 'Español',
+    translation: Es,
   },
   bg: {
-    name: "български",
-    translation: Bg
-  }
-};
+    name: 'български',
+    translation: Bg,
+  },
+  zh: {
```

Comment:
you should be able to do `"zh-CN"` in here to specify that this is simplified chinese :)

Reply:
Yes, that's what I want to.  
But when I renamed, the error occurs saying that you shouldn't use hyphen as tag name, and I'm lazy, so I changed it to `zh`.

I can hardly use the computer. If there's any spare time I'll give it another shot.

Reply:
if you want, i can merge your changes as-is and give it a shot myself to rename it to "zh-CN" :)

Reply:
Sure. If there's any bug please fix it.